src/python/pyjaia/pyjaia/drift_interpolation.py

```diff
@@ -114,26 +202,28 @@ def getInterpolatedDrifts(drifts: List[Drift], delta=50):
     outputDrifts: List[Drift] = deepcopy(drifts)
 
     if len(drifts) == 2:
-        # Just interpolate along a line
+        # If we only have two points, then just interpolate along a line
         lineString = LineString([drifts[0].location.list(), drifts[1].location.list()])
         lineLength = measurement.length(lineString, units='m')
 
-        nPoints = min(math.ceil(lineLength / delta), MAX_INTERPOLATION_POINTS)
+        # We need at least 1 point, or we get division by zero
+        nPoints = clamp(math.ceil(lineLength / resolutionDistance), 1, MAX_INTERPOLATION_POINTS)
```

Comment:
I'm a little lost here, if the number of drifts == 2, how can we have 0 points?

Reply:
Rogue comment deleted!